zoom-meeting-sdk-linux/h/meeting_service_components/meeting_webinar_interface.h

```
@@ -1,82 +1,115 @@
-/*!
-* \file meeting_webinar_interface.h
-* \brief Meeting Service Webinar Interface.
-* 
-*/
+/**
+ * @file meeting_webinar_interface.h
+ * @brief Meeting Service Webinar Interface.
+ */
 
 #ifndef _MEETING_WEBINAR_INTERFACE_H_
 #define _MEETING_WEBINAR_INTERFACE_H_
```

Clang (14.0.6)

[warning] declaration uses identifier '_MEETING_WEBINAR_INTERFACE_H_', which is a reserved identifier

(bugprone-reserved-identifier)